.github/workflows/ci.yml

```diff
@@ -39,6 +39,12 @@ jobs:
           # "online-only" test markers, since any test that's online
           # but not marked as such will fail.
           unshare --map-root-user --net make test TEST_ARGS="--skip-online"
+      - name: sign c.txt
```

Comment:
I'm just doing this to get the certificate and signature for an artifact signed with GitHub Actions. Will rebase and get rid of this before merging.